Classes/Controller/Backend/Manager/NotificationActivationController.php

```diff
@@ -27,7 +28,7 @@ class NotificationActivationController extends ManagerController
      * @param string $notificationIdentifier
      * @param string $filterEvent
```

Comment:
The default value is `null` so:

```suggestion
     * @param string|null $filterEvent
```